OpenEdXMobile/src/main/java/org/edx/mobile/http/authenticator/OauthRefreshTokenAuthenticator.java

```diff
@@ -88,6 +91,9 @@ public synchronized Request authenticate(Route route, final Response response) t
                                 .header("Authorization", currentAuth.token_type + " " + currentAuth.access_token)
                                 .build();
                     }
+                case DISABLED_USER_ERROR_MESSAGE:
```

Comment:
Can we add a doc here?
`If the user logged In through social auth & if user got disabled then on next first API call app will force the user to logout and navigate it to the splash screen`

Reply:
update to 
```
                     // If the user is logged-in & marked as disabled then on the next server response
                    // app will force the user to logout and navigate it to the launcher screen.
```